kuzzleio/sdk-go
# Deploy JAR on bintray

## What does this PR do ?

- Deploy `kuzzle-sdk-sources`, `kuzzle-sdk-javadoc` and `kuzzle-sdk` on bintray according to the architecture manually set into the `ARCH` environment variable or by the CI.
- Tries to load the libkuzzle-wrapper-java dynamically form the java.library.path, if it fails then write in the `libs` directory of the project using the sdk the embeded `libkuzzle-wrapper-java.do` then load it.

### How should this be manually tested?

- Set the `BINTRAY_USER` environment variable
- Set the `BINTRAY_KEY` environment variable
- SET the `ARCH` environment variable
Go to `/internal/wrappers/build/java` and run `gradle bintrayUpload`
See on https://bintray.com/kuzzle/maven/kuzzle if the new version of kuzzle-sdk-<ARCH> is published.
:warning: I don't recommend testing this as it will publish an official sdk on bintray. :warning: 

### Boyscout

- collection/create.go: Remove useless if
- connection/websocket/web_socket.go: Remove comment
- internal/wrappers/cgo/kuzzle/destructors.go: fix memleak
